lib/timeline/component/LineGraph.js

```diff
@@ -341,7 +341,7 @@ LineGraph.prototype._onRemove = function (ids) {
   this._onUpdate(ids);
 };
 LineGraph.prototype._onUpdateGroups = function (groupIds) {
-  this._updateAllGroupData();
+  this._updateAllGroupData(null,groupIds);
```

Comment:
I was wondering if we should run a code style formatter over all files, as it's currently a bit of a mess.